src/BTConnectionManager.cpp

```diff
@@ -459,10 +459,11 @@ QString BTConnectionManager::tailVersion() const
 
 QString BTConnectionManager::currentDeviceID() const
 {
-    if(isConnected()) {
+    // We should check for d->btControl because we may have fakeTailMode
+    if(isConnected() && d->btControl) {
         return d->btControl->remoteAddress().toString();
     }
-    return QLatin1String{};
+    return QString();
```

Comment:
Not entirely sure what this achieves... but sure

Reply:
The method `QString BTConnectionManager::currentDeviceID()` returns `QString`, not `QLatin1String`, so there is no any reason to use `QLatin1String` there. We have at least one disadvantage: when we use `QLatin1String` then we should convert `QLatin1String` to `QString`.